src/ocrd_utils/config.py

```diff
@@ -160,13 +160,18 @@ def _ocrd_download_timeout_parser(val):
 config.add("OCRD_NETWORK_CLIENT_POLLING_SLEEP",
            description="How many seconds to sleep before trying again.",
            parser=int,
-           default=(True, 30))
+           default=(True, 10))
 
 config.add("OCRD_NETWORK_CLIENT_POLLING_TIMEOUT",
            description="Timeout for a blocking ocrd network client (in seconds).",
            parser=int,
            default=(True, 3600))
 
+config.add("OCRD_NETWORK_CLIENT_POLLING_PRINT",
+           description="Whether the blocking client commands should print status output each iteration.",
+           parser=bool,
+           default=(True, False))
+
```

Comment:
I appreciate the functionality, obviously. But since you have added `print_output` kwargs, why not just have this as a parameter to `ocrd network client workflow run --print-state`? It's fine either way but I'm wondering where else polling happens and whether enabling that config env var will cause a log of logging in different places.

Reply:
That's also possible, yes. Just for usability reasons. Polling happens only on workflow jobs and processing jobs.

> I'm wondering where else polling happens and whether enabling that config env var will cause a log of logging in different places.

No, it will not have such side effects in other places, e.g., processing server integration tests. By default, the printing is always False, and the environment variable is passed only for the network client.

Reply:
[342ef3a](https://github.com/OCR-D/core/pull/1277/commits/342ef3a78f3620ff3e63200b2a9bc4c11639c581)

Reply:
Thanks for the explanation and the CLI flag